cmd/lk/token.go

```diff
@@ -19,14 +19,27 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"slices"
 	"time"
 
+	"github.com/charmbracelet/huh"
 	"github.com/urfave/cli/v3"
 
 	"github.com/livekit/protocol/auth"
 	"github.com/livekit/protocol/livekit"
 )
 
+const (
+	usageCreate   = "Token bearer can create or delete rooms"
```

Comment:
seems a bit repetitive, what about just dropping the `Token bearer can` or replace with `ability to`

```suggestion
	usageCreate   = "Token bearer can create or delete rooms"
```